gcc/testsuite/gcc.target/riscv/zc-zcf-zbb-arch.c

```diff
@@ -0,0 +1,9 @@
+/* { dg-do assemble } */
+/* { dg-options "-march=rv32i_zcb_zbb -mabi=ilp32" } */
```

Comment:
I see here that you are testing `zcb_zbb` but the name of the test is `zcf-zbb`, is that correct?

Reply:
Resolved in latest patch.